posthog/session_recordings/models/session_recording_playlist.py

```diff
@@ -43,6 +51,25 @@ class Meta:
             Index(fields=["deleted", "-last_modified_at"], name="deleted_n_last_mod_desc_idx"),
         ]
 
+    @classmethod
+    def get_file_system_unfiled(cls, team: "Team") -> QuerySet["SessionRecordingPlaylist"]:
+        base_qs = cls.objects.filter(team=team, deleted=False)
+        return cls._filter_unfiled_queryset(base_qs, team, type="replay_playlist", ref_field="short_id")
+
+    def get_file_system_representation(self) -> FileSystemRepresentation:
+        return FileSystemRepresentation(
```

Comment:
for naming... it's fine as it is

we're very likely to split them to "saved filters" and "collections" soon so the current name will go away